script/Arcueid/Day 5/05_00_ARC05_3_2.txt

```diff
@@ -217,7 +225,8 @@ But even so, she didn't veto my idea.
 [sha:41d2b4a34b1673a7d435c093c0ed99d931ca4f48]{
 -- Page 245, Offset 9199.
 -- 俺の気持ちを尊重してくれたのか、それとも勝算的には変わらないからなのか。
-I wonder if it's because she respects my feelings, or if it's because the odds wouldn't change either way.
+I wonder if it's because she respects my feelings,
+ or if it's because it won't make actually make much of a difference either way.
```

Comment:
Two makes in there, but I do feel like the structure could be better
that and 気持ちを尊重し is a bit hard to interpret like this
>Whether that's because she has some regard for how I feel, or because it won't actually make a difference either way, I don't know.